src/content/docs/en/guides/assets.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Assets (Experimental)
+description: Learn how to enable experimental asset support in Astro.
+i18nReady: false
+---
+
+**Built-in Asset Support** is enabled in Astro behind an experimental flag. This built-in feature will eventually replace the optional `@astrojs/image` integration.
+
+The new assets experience currently features:
+
+- A new built-in `<Image />` component
+- Relative images with automatic optimization in Markdown and MDX files
+- Integration with content collections
+- Improved error messages and types
+
+:::caution
+Assets are an _experimental_ Astro feature. This API is subject to change before it is marked as stable.
+:::
+
+## Enabling Assets in your Project
+
+Enabling assets may cause some breaking changes in your project. It may also require some manual changes to take advantage of new features.
+
+Please check every section below to avoid errors and to get the most out of using the experimental assets option!
+
+### Update Config
+
+To enable built-in asset support, add the following lines to your `astro.config.mjs` configuration file:
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+ experimental: {
+  assets: true
+ }
+});
+```
+
+### Move your images to `src/assets/`
+
+Create a new folder `src/assets/` which Astro will recogize as your new assets folder.
```

Comment:
Typo: `recogize` -> `recognize`
The repeating "new/folder" here feels a bit odd to me, what about rewording a bit?
```suggestion
Create an `src/assets/` folder, which Astro will recognize as your new assets folder.
```